.github/workflows/rust.yml

```diff
@@ -18,6 +18,10 @@ jobs:
           toolchain: stable
           target: wasm32-unknown-unknown
           override: true
+      - name: Add extra targets
+        # Workaround for https://github.com/actions-rs/toolchain/issues/165
+        run: |
+          rustup target add thumbv7em-none-eabi
```

Comment:
Is there no better way to do this? I do not get a lot of info out of the linked issue tbh.